### PR TITLE
Page Section Component

### PR DESCRIPTION
**Pull request by tinenhao**
image

### CODE REVIEW

File: src/components/PageSection.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
```
Comment (jason-ng-zq99) [inline src/components/PageSection.tsx:1]: I think the latest version can do without this line

**Review by jason-ng-zq99 — APPROVED** [review]
Minor suggestions; LGTM!

File: src/components/PageSection.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  }
+  return (
+    <MainContainer>
+      <ImageContainer image={props.image} />
+      <TextContainer>
+        <Title>{props.title}</Title>
+        <Body>{props.description}</Body>
+      </TextContainer>
+    </MainContainer>
+  )
```
Comment (gunbux) [inline src/components/PageSection.tsx:31]: Maybe can merge the two return statements?
```suggestion
  return (
      <MainContainer>
        {props.position !== 'right' && <ImageContainer image={props.image} />}
        <TextContainer>
          <Title>{props.title}</Title>
          <Body>{props.description}</Body>
        </TextContainer>
        {props.position === 'right' && <ImageContainer image={props.image} />}
      </MainContainer>
    )
```

**Review by gunbux — APPROVED** [review]
Minor changes but otherwise lgtm 👍

File: src/components/PageSection.tsx (hunk 1 above)
Comment (tinenhao) [reply src/components/PageSection.tsx]: fixed in commit d169cbc

File: src/components/PageSection.tsx (hunk 2 above)
Comment (tinenhao) [reply src/components/PageSection.tsx]: fixed in commit d169cbc

**Pull request merged**